web_app/src/form.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css?family=Raleway');
+@font-face{
+	font-family: 'Raleway', sans-serif;
+}
+
+.modal{
+	font-family: 'Raleway';
+	display: none;
+	position: fixed;
+	z-index: 1;
+	left: 0;
+	top: 0;
+	width: 100%;
+	height: 100%;
+	overflow: auto;
+	background-color: rgba(0,0,0,0.4);
+}
+.modal-content{
+	display: flex;
+	justify-content: center;
+}
+
+#title{
+	padding-left: 100px;
+	padding-right: 100px;
+	padding-top: 10px;
+	padding-bottom:10px;
+	text-align: center;
+	color: #fff;
+	background-color: #28493a;
+	margin:0 auto;
+}
+
+#form-body{
+	padding-top: 10px;
+	padding-bottom: 10px;
+	background-color: #fff;
+}
+
+#text-input{
+	width: 100px;
+}
+
+.desc{
+	width: 50%;
+	text-align: right;
+	margin-right: 5px;
+}
+.input-item{
+	width: 50%;
+	text-align: left;
+	margin-left: 5px;
+}
+.params{
+	width: 100%;
+	display: flex;
+	justify-content: center;
+}
+#create-button{
+	font-family: 'Raleway';
+	background-color:#93ca68;
+	color: #fff;
+	border-radius: 5px;
+	display: block;
+	margin: auto;
+}
+/*the button*/
+.close:hover,
```

Comment:
Why is this class named `close`? Is there any reason to not just style based on the `create-button` id?

Reply:
Looking at the code again we might want to separate the behavior of the `create-button` and the actual `close` button